inference-engine/tests/functional/shared_test_classes/src/single_layer/convert_color_nv12.cpp

```diff
@@ -44,7 +45,9 @@ void ConvertColorNV12LayerTest::SetUp() {
     } else {
         auto uvShape = ov::Shape{inputShape[0], inputShape[1] / 2, inputShape[2] / 2, 2};
         auto param_y = std::make_shared<ov::op::v0::Parameter>(ngPrc, inputShape);
+        param_y->set_layout(ov::Layout("NHWC"));
         auto param_uv = std::make_shared<ov::op::v0::Parameter>(ngPrc, uvShape);
+        param_uv->set_layout(ov::Layout("NHWC"));
```

Comment:
CC @nosovmik 
why do we need it?

Reply:
I haven't found another way to find out input data format

Reply:
but why do you need this information?

Reply:
According to NV12toRGB/BGR - input shape shall be always in NHWC format.
https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/image/NV12toRGB_8.md
If GPU needs such information, they can rely on this. Anyway input for this operation may not be always a 'Parameter', but some other operation which doesn't have 'layout' attribute